### ISSUE TITLE
15.10.0-beta.5 is published on npm with latest tag

### ISSUE THREAD

**Issue by michaeljota** [vuejs/vue-loader#1967]
### Version
16.8.3

### Reproduction link
[www.npmjs.com](https://www.npmjs.com/package/vue-loader)

### Steps to reproduce
run npm install vue-loader

### What is expected?
A 16.8.3 should be installed.

### What is actually happening?
15.10.0-beta.5 is installed

---
I don't know if that is intentional, but given that the "latest" version should be on the 16, I think this may be a bug. Feel free to ignore this if it is not.

**Comment by carki001** [comment]
Yes indeed, the version currently tagged as latest is 15.10.0-beta.5. I thought the latest was 17.0.0.
